pkg/annotations/annotations.go

```diff
@@ -16,6 +16,10 @@ const (
 	// DevicesAnnotation is a set of devices to give to the container
 	DevicesAnnotation = "io.kubernetes.cri-o.Devices"
 
+	// DisableDeviceOwnershipFromSecurityContextAnnotation is the K8S annotation used by kubelet/device-plugins
+	// to opt-out getting Devices UID/GID set as RunAsUser/RunAsGroup
+	DisableDeviceOwnershipFromSecurityContextAnnotation = "io.kubernetes.cri.DisableDeviceOwnershipFromSecurityContext"
+
```

Comment:
I think this should be opt-in to begin. Maybe just have:
`UseDeviceOwnershipFromSecurityContextAnnotation`.

Then the kubelet can pass that down to us, and we only interpret *if* we have that field.

that also means that a user has to specify this annotation in their list of `allowed_annotations` to have cri-o actually interpret it. This means admins will opt-into the special behavior